### PR TITLE
[WFLY-16129] [allowlist/blocklist] Problematic Language usage depreca…

### PR DESCRIPTION
**Pull request by moulalis**
Issue: https://issues.redhat.com/browse/WFLY-16129

### CODE REVIEW

File: docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_JWT.adoc [hunk 1]
```diff
@@ -128,7 +128,7 @@ The SmallRye JWT specific properties allow for a lot of customisation not covere
 |60
 |The refresh inerval where `mp.jwt.verify.publickey.location` points to a HTTPS location.
 
-|smallrye.jwt.whitelist.algorithms
+|smallrye.jwt.blocklist.algorithms
```
Comment (darranl) [inline docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_JWT.adoc:131]: @moulalis Has the JWT code actually changed?

**Review by darranl — CHANGES_REQUESTED** [review]
We need to verify what properties SmallRye JWT is using.

**Comment by moulalis** [comment]
> smallrye.jwt.whitelist.algorithms

@darranl As per https://github.com/smallrye/smallrye-jwt/blob/3.1.1/doc/modules/ROOT/pages/configuration.adoc, it doesn't mention any "whitelist" property, instead they have smallrye.jwt.verify.algorithm.  The whitelist prop is removed https://github.com/smallrye/smallrye-jwt/commit/77eab5054915342b8ee9e121c5a7e4a7b94f74e1 here.

**Comment by bstansberry** [comment]
@moulalis @darranl I recommend that this 'smallrye.jwt.blocklist.algorithms' entry in the table be deleted and we file a separate JIRA to sync this doc page with the current set of smallrye config properties, as there are a number of differences, plus some of the smallrye-specific ones are documented as deprecated with an mp.* one recommended. That work is unrelated to problematic language though.

The string 'whitelist' does not appear in the smallrye-jwt 3.1.1 source, so it seems pretty clear the entry in the docs table here is invalid.

**Comment by yersan** [comment]
Hi, @moulalis would you mind following Brian's recommendation and deleting the entry from the doc?

Just for the record, I've just created this Jira to track the documentation changes for JWT: https://issues.redhat.com/browse/WFLY-16382

**Comment by moulalis** [comment]
> Hi, @moulalis would you mind following Brian's recommendation and deleting the entry from the doc?
> 
> Just for the record, I've just created this Jira to track the documentation changes for JWT: https://issues.redhat.com/browse/WFLY-16382

PR Updated. Thanks.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**